tsconfig.json

```diff
@@ -12,11 +12,11 @@
     // "declarationMap": true,                /* Generates a sourcemap for each corresponding '.d.ts' file. */
     "sourceMap": true /* Generates corresponding '.map' file. */,
     // "outFile": "./",                       /* Concatenate and emit output to single file. */
-    "outDir": "/dev/null" /* Redirect output structure to the directory. */,
+    // "outDir": "./build" /* Redirect output structure to the directory. */,
     // "rootDir": "./src",                    /* Specify the root directory of input files. Use to control the output directory structure with --outDir. */
     // "composite": true,                     /* Enable project compilation */
     // "removeComments": true,                /* Do not emit comments to output. */
-    // "noEmit": true,                        /* Do not emit outputs. */
+    "noEmit": true /* Do not emit outputs. */,
```

Comment:
Oops, this line was totally messing with me! I kept running build and deploy (firebase functions) but no new output was being generated, so the same code from `lib` folder was being deployed over and over, but I didn't realize it. I was quite confused.

Reply:
@sstur Oops, sorry about that.

So what do you think about it? Should we:
1. Change the default into `false`
2. Just comment it again
3. Let it be but make sure to overwrite it from local config/rule when needed

cc @darcien

Reply:
I think option 2 is preferable.
Right now we mostly use this config for RN project, which don't need the output file.